### PR TITLE
Add OntologyRaw

### PR DESCRIPTION
**Pull request by EivindYN**

### CODE REVIEW

**Review by sebastianvitterso — APPROVED** [review]
Looks useful!

File: src/main/java/services/dataclasses/OntologyRaw.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package services.dataclasses;
+
+public class OntologyRaw {
```
Comment (sebastianvitterso) [inline src/main/java/services/dataclasses/OntologyRaw.java:3]: Should this class have a function like this?
```java
public OWLOntology toOWLOntology() {
  // return this object converted to an OWLOntology-object
}
```

Comment (EivindYN) [reply src/main/java/services/dataclasses/OntologyRaw.java:6]: Added it!

**Review by sebastianvitterso — APPROVED** [review]
Good, good!

**Pull request merged**